src/main/webapp/css/user-page.css

```diff
@@ -14,39 +14,90 @@
  * limitations under the License.
  */
 
-/* Set the size of the message input to be 100 pixels high
-   and will fill the width of its parent element. */
-#message-input {
-  height: 100px;
+
+.about-me-container {
+  background: white;
+	text-align: center;
+	width: 100%;
+}
+
+.about-me-input {
+  border: none;
+  height: 110px;
+  width: 99.4%;
```

Comment:
why 99.4? seems so specific

Reply:
Real pain figuring out this number, but I think it fits the best